### PR TITLE
fix(ROIThresholdsTools): Small fixes for Rectangle and Circle ROIStartEndThresholds tools

### PR DESCRIPTION
**Pull request by Celian-abd**
### Context
A few fixes for the previous PR that was merged https://github.com/cornerstonejs/cornerstone3D/pull/1128

### Changes & Results
- Fixed the calculation of the projection points
- Fixed the handles so we don't have to modify and calculate them in OHIF anymore
- Update the pointsInsideVolume when we modify the start or end
- Fixed the name in the CircleROIStartEndThreshold example

### Testing
circleROIStartEndThresholdWithSegmentation and rectangleROIStartEndThresholdWithSegmentation examples

#### PR

- [] My Pull Request title is descriptive, accurate and follows the
  semantic-release format and guidelines.

#### Code

- [] My code has been well-documented (function documentation, inline comments,
  etc.)

- [] I have run the `yarn build:update-api` to update the API documentation, and have
  committed the changes to this PR. (Read more here https://www.cornerstonejs.org/docs/contribute/update-api)

#### Public Documentation Updates

- [] The documentation page has been updated as necessary for any public API
  additions or removals.

#### Tested Environment

- [] "OS: 
- [] "Node version: 
- [] "Browser:
  

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

File: packages/tools/src/tools/segmentation/CircleROIStartEndThresholdTool.ts [hunk 1]
```diff
@@ -340,6 +328,7 @@ class CircleROIStartEndThresholdTool extends CircleROITool {
       const radius = getCanvasCircleRadius(canvasCoordinates);
       const { centerPointRadius } = this.configuration;
       const canvasCorners = getCanvasCircleCorners(canvasCoordinates);
+      const canvasCorners = getCanvasCircleCorners(canvasCoordinates);
```
Comment (jlopes90) [inline packages/tools/src/tools/segmentation/CircleROIStartEndThresholdTool.ts:331]: is duplicated

**Review by jlopes90 — APPROVED** [review]

**Comment by sedghi** [comment]
can you revert your changes of uuid4

File: packages/tools/src/tools/segmentation/CircleROIStartEndThresholdTool.ts [hunk 2]
```diff
@@ -353,12 +341,14 @@ class CircleROIStartEndThresholdTool extends CircleROITool {
...
         );
+        data.startCoordinate = startCoord;
       }
       if (Array.isArray(endCoordinate)) {
         endCoord = this._getCoordinateForViewplaneNormal(
           endCoord,
           viewplaneNormal
         );
+        data.endCoordinate = endCoord;
       }
```
Comment (sedghi) [inline packages/tools/src/tools/segmentation/CircleROIStartEndThresholdTool.ts:352]: Can we stick to `startCoordinate` and `endCoordinate` perhaps? I don't like the direction this is going, with `startCoord` and `startCoordinate`, etc.

**Comment by salimkanoun** [comment]
@sedghi this PR should be ready, is there anything celian could do (he is going to leave Pixilib for 6 months so we would want to close this project)

**Review by sedghi — APPROVED** [review]

**Pull request merged**